airbyte-workers/src/main/java/io/airbyte/workers/process/KubePodProcess.java

```diff
@@ -538,8 +536,13 @@ private void close() {
     Exceptions.swallow(this.stdoutServerSocket::close);
     Exceptions.swallow(this.stderrServerSocket::close);
     Exceptions.swallow(this.executorService::shutdownNow);
-    Exceptions.swallow(() -> portReleaser.accept(stdoutLocalPort));
-    Exceptions.swallow(() -> portReleaser.accept(stderrLocalPort));
+    try {
+      portReleaser.accept(stdoutLocalPort);
+      portReleaser.accept(stderrLocalPort);
+    } catch (Exception e) {
+      LOGGER.error("Error releasing ports ", e);
```

Comment:
should this rethrow as runtime?

Reply:
I don't think so. Since we are here, the data is already synced so failing the job doesn't add much. I think logging as an error is the best way since it makes it clear why this doesn't work later on.

Reply:
I made the error message clearer to show this doesn't affect the data being synced, but will affect future syncs if ports leak.